### ISSUE TITLE
Remove redundant "By" in mod action messages

### ISSUE THREAD

**Issue by teradw** [CBenni/mt2#99]
Actions other than (un)ban/timeout

![image](https://user-images.githubusercontent.com/24725862/39300245-561571b8-494b-11e8-8520-cbcef49fe78e.png)

Or maybe just append `(By: user)` part to the original IRC message instead of displaying 2 lines for same event?

**Comment by mattts1** [comment]
Yeah it does make sense just to append (By tera_) at the end of "This room is no longer in followers-only mode" instead of two separate messages, because it would be obvious at that point that tera_ used the command.
